src/components/Hero.jsx

```diff
@@ -1,17 +1,126 @@
 import React from 'react'
+import styled from 'styled-components'
 
-const Hero = ({ eyebrowText, mainHeading, content, img, imgDescription }) => {
+const StyledHeroSection = styled.section`
+  background-color: #b3ddf2;
+
+  @media (min-width: 769px) {
+    display: flex;
+    align-items: center;
+    padding-left: 3rem;
+    gap: 1.5rem;
+  }
+`
+
+const StyledContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+  padding: 3rem 1.5rem;
+
+  @media (min-width: 769px) {
+    align-items: flex-start;
+    padding: 4rem 0px;
+    gap: 1.5rem;
+    width: 499px;
+    height: 330px;
+  }
+  @media (min-width: 1200px) {
+    width: 540px;
+  }
+`
+
+const StyledEyebrowText = styled.span`
+  font-style: normal;
+  font-weight: 600;
+  letter-spacing: 0.05em;
+  line-height: 21px;
+  text-align: center;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+  height: 21px;
+  color: #001426;
+
+  @media (min-width: 649px) {
+    font-size: 1.125rem;
+  }
+`
+const StyledHeading = styled.h2`
+  text-align: center;
+  font-size: 2rem;
+  font-weight: 700;
+  color: #001426;
+  flex: none;
+
+  @media (min-width: 649px) {
+    font-size: 2.625rem;
+  }
+  @media (min-width: 769px) {
+    text-align: left;
+  }
+`
+
+const StyledHeroImageContainer = styled.div`
+  @media (min-width: 769px) {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    align-items: flex-start;
+    padding: 0px;
```

Comment:
For 0, there's no need to add px (and we should definitely be using rem anyway). But 0 is fine enough for measurements.